apps/condo/domains/banking/access/BankAccount.js

```diff
@@ -53,18 +51,6 @@ async function canReadBankAccounts ({ authentication: { item: user } }) {
         })
     }
 
-    if (user.type === RESIDENT) {
-        const residents = await find('Resident', { user: { id: user.id } })
-        if (residents.length > 0) {
-            const propertyIds = uniq(map(residents, 'property'))
-            conditions.OR.push({
-                property: {
-                    id_in: propertyIds,
-                },
-            })
-        }
-    }
-
     return conditions
```

Comment:
does this mean that any resident will get empty conditions here, like support/admin ?

Reply:
Yep, that should resolve to empty filter, which will cause reading everything...